infra/cifuzz/fuzz_target.py

```diff
@@ -134,3 +186,67 @@ def get_test_case(self, error_string):
     if match:
       return os.path.join(self.out_dir, match.group(1))
     return None
+
+  def get_lastest_build_version(self):
+    """Gets the latest OSS-Fuzz build version for a projects' fuzzers.
+
+    Returns:
+      A string with the latest build version or None.
+    """
+    if not self.project_name:
+      return None
+    sanitizer = 'address'
+    version = '{project_name}-{sanitizer}-latest.version'.format(
+        project_name=self.project_name, sanitizer=sanitizer)
+    version_url = url_join(GCS_BASE_URL, self.project_name, version)
+    try:
+      response = urllib.request.urlopen(version_url)
+    except urllib.error.HTTPError:
+      logging.error(
+          'Error getting the lastest build version for %s from url %s.',
+          self.project_name, version_url)
+      return None
+    return response.read().decode('UTF-8')
```

Comment:
nit: no need for 'UTF-8'. That's the default in Python 3 (https://docs.python.org/3/library/stdtypes.html#bytes.decode)

Reply:
Done.